compiler/src/iree/compiler/Codegen/LLVMGPU/test/set_transform_strategy_matmul.mlir

```diff
@@ -1,4 +1,5 @@
-// RUN: iree-opt %s --split-input-file --pass-pipeline="builtin.module(hal.executable(hal.executable.variant(iree-llvmgpu-select-lowering-strategy)))" --iree-codegen-llvmgpu-enable-transform-dialect-aligned-matmul | FileCheck %s
+// RUN: iree-opt %s --split-input-file --pass-pipeline="builtin.module(hal.executable(hal.executable.variant(iree-llvmgpu-select-lowering-strategy)))" \
+// RUN:   --iree-codegen-llvmgpu-enable-transform-dialect-aligned-matmul | FileCheck %s
```

Comment:
Why is the flag on a separate line? Does it actually matter?

Reply:
Makes it easier to read/edit. The project-wide line column limit is 80, we are more lenient in tests because there's no mlir-format.